arches/app/media/js/views/components/search/base-search-view.js

```diff
@@ -50,8 +50,9 @@ define([
         doQuery: function() {
             const queryObj = JSON.parse(this.queryString());
             if (self.updateRequest) { self.updateRequest.abort(); }
+            const requestMethod = JSON.stringify(queryObj).length > 1800 ? "POST" : "GET";
```

Comment:
where did this number come from?  2048 seems like the most Edge can handle but most other browsers much more.

Reply:
This leaves a generous buffer in case the host is really long as can sometimes be the case with development hosts using Azure, for example: "https://longsubdomain.azure.cloudapp.westus3.developmentserverlongname." Is there a reason you think we should increase this limit + forego compatibility with Edge?